doc/filter_tutorial.md

```diff
@@ -0,0 +1,376 @@
+First a summary of what you’ll do:
+- Install MongoDB Compass
+- Connect to cluster with Rubin alerts
+- Create your filter
+- Save your candidates
+- Explore your results
+
+### 0. Note
+I based most of this work on the tutorial you can find here, so, if you have any issues, I would start looking them up there.
+### 1. To install MongoDB Compass
+ Go to https://www.mongodb.com/products/compass and click in ‘Download now’
+### 2. Connecting to cluster
+Once Mongo DB Compass is installed, launch it, and connect to the cluster by
+
+   - Opening Mongo DB Compass
+   - Copy the following into the box under New Connection, and click Connect.  <br/>
+mongodb+srv://alert_digger:WhereIsTheKN@cluster0.s0ada.mongodb.net/test  <br/>
+
+
+     ![](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial1.png)
+
+
+
+ - To see the alerts (around 4000 of them) navigate in the left menu to the folder ZTF > lsst_800
+
+   ![This is an image](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial2.png)
+
+
+
+ **Tab description**
+
+- Documents: here you can see all the avro packages, and their individual properties. You can add Data or export a collection. You can apply filters here too, but it is easier in the Aggregation tab. It displays only 20 alerts.
+- Aggregations: Here you can make a new filter and apply it to the collection. After each step, you can visualize how many alerts passed the filters and modify the filter step to check how things change. You can export the results and resulting filter.
+- Scheme: tbd
+- Explain plan: it shows the details of your filter
+- Indexes: tbd
+- Validation: tbd
+
+### 3. To make a new filter
+In the Aggregation tab, click on Create New > Pipeline from text. This will promt a new window where you can write/paste your filter.
+
+ ![This is an image](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial3.png)
+
+
+At the end of this document there is a filtering example, used by the MMA group to vet SGRB candidates. Once you plug in your filter, MongoDB Compass will break it down for you in different steps. This filter has two $project stages before the $match stage. You can play around with these to modify your filter.
+
+ ![This is an image](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial4.png)
+
+
+
+### 4. Saving candidates
+Once you are done with the filtering, you can
+<ol type="a">
+  <li>  ‘Export to language’ (recommended, as MongoDB Compass is not good for saving your filters) - button next to ‘Create new +’ </li>
+    <li>  Save your current filter – Save button next to ‘Create new +’ </li>
+    <li>  Export your results as a json or csv file – click on the export button next to ‘Run’ and then choose your preferred format. </li>
+  </ol>
```

Comment:
Why are there HTML tags suddenly?

Reply:
I couldn't find a way to use letters instead of numbers on MD, and this worked fine. I changed to numbers to be consistent